connectors/tests/ftest.sh

```diff
@@ -7,26 +7,39 @@ PERF8=$2
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 ROOT_DIR="$SCRIPT_DIR/../.."
+PLATFORM='unknown'
+unamestr=$(uname)
+if [[ "$unamestr" == 'Linux' ]]; then
+   PLATFORM='linux'
+elif [[ "$unamestr" == 'Darwin' ]]; then
+   PLATFORM='darwin'
+fi
+
 
 cd $ROOT_DIR/connectors/sources/tests/fixtures/$NAME
 make run-stack
+
 # XXX make run-stack should be blocking until everythign is up and running by checking hbs
 sleep 30
 
 $ROOT_DIR/bin/fake-kibana --index-name search-$NAME --service-type $NAME --debug
 
-make load-data
+#make load-data
 if [[ $PERF8 == "yes" ]]
 then
-    $ROOT_DIR/bin/perf8 --memray --pyspy --psutil -c $ROOT_DIR/bin/elastic-ingest --one-sync --sync-now --debug
-
+    if [[ $PLATFORM == "darwin" ]]
```

Comment:
The description says
> don't run pyspy on darwin

but this _only_ runs pyspy on darwin. Should this be a `-ne`?

Reply:
ahah good catch thanks :)